sites/next.skeleton.dev/src/examples/tailwind/forms/ExampleGroups.astro

```diff
@@ -1,14 +1,14 @@
 ---
-import { CircleDollarSign, Check, Search } from 'lucide-react';
+import { CircleDollarSign, Check, Search } from 'lucide-svelte';
```

Comment:
@phamduylong we actually default to the React icons for any Astro-based components. They have a slightly better tree shaking solution than Svelte. If you don't mind, go ahead and revert this change.

Additionally, if you are ever using Svelte icons within the documentation (say in a .svelte component), make sure you import these per the guidelines here. Otherwise these will not be tree shaken properly:

https://next.skeleton.dev/docs/integrations/iconography/svelte#usage

Essentially you import the icons one-by-one with multiple dedicated import statements.